unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/Tests/WorldChatWindowViewMock.cs

```diff
@@ -9,7 +9,8 @@ public class WorldChatWindowViewMock : MonoBehaviour, IWorldChatWindowView
     public event Action<string> OnOpenPublicChannel;
     public event Action<string> OnUnfriend;
     public event Action<string> OnSearchChannelRequested;
-    public event Action OnRequireMorePrivateChats;
+    public event Action OnScrollDownToTheBottom;
```

Comment:
i would remain the old name of the event `OnRequireMorePrivateChats` since the change from a button to a scroll its not important to the controller

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2484/commits/a99f44e0b3bcbf5e415f1ac6bfa636d9732a817b